zap/src/main/java/org/zaproxy/zap/extension/pscan/PluginPassiveScanner.java

```diff
@@ -401,6 +402,16 @@ protected void addTag(String tag) {
         parent.addTag(tag);
     }
 
+    /**
+     * Gets the tags attached to the alerts raised by this plugin.
```

Comment:
```suggestion
     * Gets the tags attached to the alerts raised by this plugin.
     * Can be overridden by scan rules to return the associated alert tags.
```

Or something like that?